crates/holochain_zome_types/Cargo.toml

```diff
@@ -15,6 +15,7 @@ edition = "2018"
 holo_hash = { path = "../holo_hash" }
 holochain_serialized_bytes = "=0.0.47"
 paste = "=1.0.3"
+chrono = "0.4.6"
```

Comment:
@thedavidmeister - I like this PR in general - this line will pull chrono into WASM, think that is acceptable?

Reply:
Yes, that is probably the biggest question.  Presently all Zome logging includes a lot of Timestamps as eg. [1234566345,134421343], making the logging not very useful -- this would buy us at least logging that is more usable.  So, that alone might be a worthwhile investment of the compiled Zome space required.

I think most non-trivial Zomes will deal in times, so it'll usually be a wash.

Reply:
Furthermore, this dramatically cleans up the usage of the chrono DateTime in dealing with Timestamps and Durations.  Presently, they are *very* ... broken.  It is super easy to construct Timestamps that just panic in the DateTime module (fixt did it a lot).  That is all solved with this implementation (I have exhaustive tests in my mutualcredit Zome, when we can move into holochain if we wish.)

Reply:
Just working on rebasing this forward to current develop...

Reply:
yes i'm happy with the size changes, seems the compiler is good about eliding it if you're not using it, and if you're using it then you're using it :)